mkosi

```diff
@@ -3498,6 +3512,34 @@ def run_qemu(args):
 
     os.execvp(cmdline[0], cmdline)
 
+def expand_paths(input_paths):
+    if not input_paths:
+        return []
+
+    environ = os.environ.copy()
+    # Add a fake SUDO_HOME variable to allow non-root users specify
+    # paths in their home when using mkosi via sudo.
+    sudo_user = os.getenv("SUDO_USER")
+    if sudo_user:
+        environ["SUDO_HOME"] = os.path.expanduser("~{}".format(sudo_user))
```

Comment:
i think we should leave SUDO_HOME set the way it is if it already is set, and only add it if it isn't...

Reply:
Done.